reference/3.0/Microsoft.PowerShell.Core/About/about_CommonParameters.md

```diff
@@ -507,18 +550,10 @@ Mode                LastWriteTime     Length Name
 ----                -------------     ------ ----
```

Comment:
if you would rather i file an issue for it and fix it separately, that's fine. But this is definitely broken and should be fixed one way or the other since it renders in a misleading manner.